zulipterminal/model.py

```diff
@@ -714,6 +714,7 @@ def update_rendered_view(self, msg_id: int) -> None:
                 msg_pos = self.msg_list.log.index(msg_w)
                 self.msg_list.log[msg_pos] = new_msg_w
                 self.controller.update_screen()
+                return
```

Comment:
This first commit is mainly tidying/optimizing, I assume?

If we're adding this line, can we make use of this tidying commit to also merged the `else` clause above into this code?

This commit title seems a little unclear (update_message?)